AppDevAnnouncements/Classes/NotificationViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  NotificationViewController.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 3/11/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol NotificationDelegate: class {
+    func present(_ notificationViewController: NotificationViewController)
+}
+
+class NotificationViewController: UIViewController {
+
+    private var notificationView: NotificationView!
+
+    private var announcement: Announcement
+    private weak var delegate : NotificationDelegate?
+
+    init(announcement: Announcement, delegate: NotificationDelegate) {
+        self.announcement = announcement
+        self.delegate = delegate
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .clear
+
+        notificationView = NotificationView(announcement: announcement, dismissFunc: #selector(dismissNotification), actionFunc: #selector(performCTA), target: self)
+        notificationView.layer.cornerRadius = 10
+        notificationView.clipsToBounds = true
+        view.addSubview(notificationView)
+
+        setupConstraints()
+        presentNotification()
+    }
+
+    enum Constants {
+        static let notificationViewWidth: CGFloat = 327
+    }
```

Comment:
move this to. the top of the class w/ the rest of the variables (i.e. after line 21)